src/gui/screen_bed_leveling.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "gui.hpp"
+#include "status_footer.hpp"
+#include "window_text.hpp"
+#include "window_icon.hpp"
+#include "window_term.hpp"
+#include "window_progress.hpp"
+#include "screen.hpp"
+#include "ScreenHandler.hpp"
+
+class screen_bed_leveling_t : public AddSuperWindow<screen_t> {
+    enum class state_t {
+        idle = 0,
+        homing,
+        leveling,
+        done,
+        error
+    };
+    state_t state { state_t::idle };
+
+    /** While an instance exists, MenuTimeout will be disabled.
+    **/
+    class MenuTimeoutBlocker {
```

Comment:
Just call `DisableMenuTimeout()` in constructor. and it will disable timeout for this screen. This object is not needed.

With `Screens::Access()` you get pinter to current screen

Reply:
Removed, switched to DisableMenuTimeout()